test/rng_testsuite/template_tests/subtract_with_carry_std_template_test.pass.cpp

```diff
@@ -340,7 +340,7 @@ int main() {
     EXPECT_TRUE(!err, "Test FAILED");
 #endif // #if defined(_ONEDPL_RNG_DETAILED_TESTING_)
 
-#endif // TEST_DPCPP_BACKEND_PRESENT
+#endif // TEST_DPCPP_BACKEND_PRESENT && __SYCL_UNNAMED_LAMBDA__
 
     return TestUtils::done(TEST_DPCPP_BACKEND_PRESENT);
```

Comment:
`TEST_DPCPP_BACKEND_PRESENT` -> `TEST_DPCPP_BACKEND_PRESENT && __SYCL_UNNAMED_LAMBDA__` here and for other tests

Reply:
changed